src/data/markdown/docs/30 xk6-browser/01 xk6-browser/01 Browser/newcontext--options--.md

```diff
@@ -23,34 +23,34 @@ You can customize the creation of a new browser context using the following opti
-| viewport                                                            | object  | Sets a viewport size for all pages in the context. `null` disables the default viewport. See: [viewport](#viewport-options) for the options. Defaults to `1280x720`.                                                                                                                                                                                                                     |
+| State | Option                                                              | Type    | Description                                                                                                                                                                                                                                                                                                                                                                              |
+| - | ------------------------------------------------------------------- | ------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| <span title="Not implemented">❌</span> | acceptDownloads             | boolean | Whether to automatically download files. Defaults to `true`. Follow the progress in [issue #98](https://github.com/grafana/xk6-browser/issues/98). |
+| <span title="Not implemented">❌</span> | baseURL                     | string  | The base URL to use for all relative URLs. For example: `/open-source/` will be converted to `https://k6.io/open-source/` if `baseURL` is set to `https://k6.io`. Follow the progress in [issue #432](https://github.com/grafana/xk6-browser/issues/432). |
+| ✅ | bypassCSP                                                           | boolean | Whether to bypass a page's Content-Security-Policy. |
+| ✅ | colorScheme                                                         | string  | Whether to display a page in dark or light mode by emulating the 'prefers-colors-scheme' media feature. It can be one of `'light'`, `'dark'`, `'no-preference'`. See [page.emulateMedia](/javascript-api/xk6-browser/page#page-emulatemedia-options) for the options and the [example](https://github.com/grafana/xk6-browser/blob/main/examples/colorscheme.js). Defaults to `'light'`. |
+| <span title="Work in progress">🚧</span> | deviceScaleFactor          | number  | Sets the resolution ratio in physical pixels to the resolution in CSS pixels. See the [link](https://github.com/grafana/xk6-browser/blob/main/examples/device_emulation.js) as an example. Defaults to `1`. Follow the progress in [issue #433](https://github.com/grafana/xk6-browser/issues/433). |
```

Comment:
```suggestion
| <span title="Work in progress">🚧</span> | deviceScaleFactor          | number  | Sets the resolution ratio in physical pixels to the resolution in CSS pixels. See an [example](https://github.com/grafana/xk6-browser/blob/main/examples/device_emulation.js). Defaults to `1`. Follow the progress in [issue #433](https://github.com/grafana/xk6-browser/issues/433). |
```

Though I don't think we should link to examples on GitHub. If we want to showcase something, we should do it inline on this page.

Reply:
It feels a bit strange to have examples for two of the options and not all. Either we have it for all of them or none of them. WDYT?

Also, how would i inline an example in this table? I can add them to the bottom of the page?

Reply:
> It feels a bit strange to have examples for two of the options and not all.

Hhmm true. I guess we don't need to take an absolutist stance on this (all or nothing). It's fine if we have some simple and some more advanced examples, e.g. to document possibly tricky behavior. So we might as well reuse the examples we have currently. My point was that we shouldn't link to GitHub for this. The documentation should be standalone, even if it introduces duplication we'll need to keep in sync with the code.

> how would i inline an example in this table? I can add them to the bottom of the page?

Yeah, we usually have an "Examples" section at the bottom, and you can preamble each example with something that describes it. See how this is done on other pages, and let's be consistent about it.